Adds the Chocolatey package as a way to install Hakuneko

Xav83 opened the pull request:
Hi @Hakuneko_devs,

I have created this Pull Request because I have resolved this [request for a Chocolatey package](https://github.com/chocolatey-community/chocolatey-package-requests/issues/1307) and now, we can install **hakuneko** using [Chocolatey](https://community.chocolatey.org/packages/hakunekonightly).

For now, only the latest version (`8.3.4`) is available on Chocolatey. When new versions will be released, they will, then, be added to the Chocolatey package for users to enjoy. :slightly_smiling_face:

Xav83 committed:
Adds the Chocolatey package as a way to install Hakuneko

Adds the Chocolatey package in the list of possibilities to install Hakuneko.

27136773 commented on README.asciidoc:
```diff
@@ -22,6 +22,7 @@ image:https://hakuneko.download/assets/images/discord-small.png[link=https://dis
 
 image:https://img.shields.io/github/downloads/manga-download/hakuneko/latest/total?label=HakuNeko%20%28Stable%29&logo=github[link=https://github.com/manga-download/hakuneko/releases/latest, title="Download the latest stable release of HakuNeko", window="_blank"]
 image:https://img.shields.io/github/downloads-pre/manga-download/hakuneko/latest/total?color=blue&label=HakuNeko%20%28Nightly%29&logo=azure-devops[link=https://github.com/manga-download/hakuneko/releases, title="Download the latest pre-release (nightly build) of HakuNeko", window="_blank"]
+image:https://img.shields.io/chocolatey/dt/hakunekonightly?color=blue&label=Chocolatey%20package[Chocolatey]
```
May set a link to the corresponding repository, e.g.
https://community.chocolatey.org/packages/hakunekonightly

Xav83 replied:
Great idea @ronny1982 ! I have added a commit in order to set the link :wink:

Xav83 committed:
Makes the Choclatey badge redirect to the package 

Modify the new Chocolatey badge so that it can redirect to the page of the Chocolatey package.
